README.md

```
@@ -88,3 +88,44 @@
 #### Test
 
 - [x] Storybook 테스트
+
+## step2
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)